azure-pipelines.yml

```diff
@@ -0,0 +1,35 @@
+---
+jobs:
+  - job: Linux
+    pool:
+      vmImage: 'Ubuntu 16.04'
+    steps:
+      - script: git submodule update --init -- es/upstream
+      - task: Gradle@2
+        inputs:
+          workingDirectory: ''
+          gradleWrapperFile: 'gradlew'
+          gradleOptions: '-Xmx3072m'
+          javaHomeOption: 'JDKVersion'
+          jdkVersionOption: '1.10'
+          jdkArchitectureOption: 'x64'
+          publishJUnitResults: true
+          testResultsFiles: '**/TEST-*.xml'
+          tasks: ':sql:test -PtestForks=2'
```

Comment:
why not include the tests from other modules?

Reply:
We limited travis and appveyor to run only `sql` as otherwise the tests took too long there and were terminated.
Azure pipelines would allow longer test runtimes, so we could run all. Not sure if it's worth it. We cover the other modules with jenkins. `sql` has the most randomness/flakyness.